src/libraries/System.Private.CoreLib/src/System/UInt64.cs

```diff
@@ -358,6 +358,9 @@ bool IBinaryInteger<ulong>.TryWriteLittleEndian(Span<byte> destination, out int
         // IBinaryNumber
         //
 
+        /// <inheritdoc cref="IBinaryNumber{TSelf}.AllBitsSet" />
+        static ulong IBinaryNumber<ulong>.AllBitsSet => 0xFFFFFFFFFFFFFFFF;
```

Comment:
```suggestion
        static ulong IBinaryNumber<ulong>.AllBitsSet => 0xFFFF_FFFF_FFFF_FFFF;
```

or same suggestion as above